west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 01d3f9bba6f9f36374014915e9afbb4cb28d232c
+      revision: pull/1326/head
```

Comment:
SHA to update: `328411d85839340607fe033fa4de4f59cb2eb038`

Reply:
I updated sdk-zephyr revision to 328411d85839340607fe033fa4de4f59cb2eb038